src/main/content/_assets/js/builds.js

```diff
@@ -798,7 +788,15 @@ function validate_starter_inputs(event) {
             .text();
             for(var i=0; i<options.length; i++) {
                 if(options[i].value === EEVersion) {
-                    $(options[i]).prop('selected', true);
+                    if(mpVersionValue !== "None"){
```

Comment:
```suggestion
                    if(mpVersionValue !== "None") {
```

For code consistency